pkg/agent/openflow/pipeline_windows.go

```diff
@@ -140,3 +140,96 @@ func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, r
 	}
 	return []binding.Flow{c.l3FwdFlowToRemoteViaGW(localGatewayMAC, *peerPodCIDR, category)}
 }
+
+func (c *client) connectionTrackFlowsAntreaProxyClusterIPSvcHost(category cookie.Category) []binding.Flow {
+	var flows []binding.Flow
+
+	connectionTrackCommitTable := c.pipeline[conntrackCommitTable]
+	if c.enableProxy {
+		// These 2 tables are for traffic SNAT of ClusterIP Service from Windows host.
+		serviceConnectionTrackTable := c.pipeline[serviceConntrackTable]
+		serviceConnectionTrackCommitTable := c.pipeline[serviceConntrackCommitTable]
+
+		flows = append(flows,
+			c.pipeline[spoofGuardTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
+				MatchInPort(config.HostGatewayOFPort).MatchDstIP(config.ServiceGWHairpinIPv4).
+				MatchDstMAC(globalVirtualMAC).
+				Action().GotoTable(serviceConntrackTable).
+				Cookie(c.cookieAllocator.Request(category).Raw()).
+				Done(),
+			serviceConnectionTrackTable.BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolIP).
+				Action().CT(false, serviceConnectionTrackTable.GetNext(), SNATCtZone).NAT().CTDone().
+				Cookie(c.cookieAllocator.Request(category).Raw()).
+				Done(),
+			connectionTrackCommitTable.BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
+				Cookie(c.cookieAllocator.Request(category).Raw()).
+				MatchCTMark(ServiceCTMark).
+				MatchRegMark(FromGatewayRegMark).
+				Action().GotoTable(serviceConntrackCommitTable).
+				Cookie(c.cookieAllocator.Request(category).Raw()).
+				Done(),
+			connectionTrackCommitTable.BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolIP).
```

Comment:
Since connectionTrackCommitTable is the same as serviceConnectionTrackCommitTable, why you want to differentiate the two tables in this function? Could you add comments in this function about how you want to leverage the ct to control the connection?

Reply:
Added comments.